src/schema/v2/me/createCollectionMutation.ts

```diff
@@ -56,6 +58,7 @@ export const createCollectionMutation = mutationWithClientMutationId<
   description: "Create a collection",
   inputFields: {
     name: { type: new GraphQLNonNull(GraphQLString) },
+    shareableWithPartners: { type: GraphQLBoolean },
```

Comment:
Should this also be `new GraphQLNonNull(GraphQLBoolean)` so it's non nullable?

Reply:
Do we want to make this a required field? In Gravity it's optional, so I reflected that logic here.